apps/comps/app/competitions/page.tsx

```diff
@@ -1,7 +1,42 @@
-export default function Competitions() {
+import React from "react";
+
+import { AgentSpotlightSection } from "../../components/agent-spotlight-section";
```

Comment:
Not a big deal, but I prefer to use the `@/` alias we set up in order to make imports more portable and readable.